web3-react/src/libs/components/WalletConnectOption.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import {
+  getConnection,
+  ConnectionType,
+  tryActivateConnector,
+  tryDeactivateConnector,
+} from '../connections'
+
+export const WalletConnectOption = ({
```

Comment:
Now that I think about it, can we build this Option component in a reusable way so that it accepts a ConnectionType and we don't need 3 separate files for each connection? I believe this is what we already do in web3-react example, where we have a generic component that takes a provider.

Reply:
Awesome call, thanks!